### PR TITLE
PP-16 footer

### PR DESCRIPTION
**Pull request by arigakova**
created footer page

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,10 +24,22 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-  </body>
+        <link
```
Comment (AlonaVladymyrovaTrinity) [inline public/index.html:27]: Why were added two libraries of different versions of bootstrap?

File: src/component/layout/About/About.js
Comment (AlonaVladymyrovaTrinity) [inline src/component/layout/About/About.js:1]: All changes have been diligently implemented and thoroughly considered, taking into account the discussions on enhancing this page.

File: public/index.html (hunk 1 above)
Comment (arigakova) [reply public/index.html:27]: Good catch, one of the version was removed

**Review by AlonaVladymyrovaTrinity — APPROVED** [review]
overall looks good

**Review by badgerbadgers — APPROVED** [review]
LGTM! Bootstrap should make creating components and styling easier for the group now!

**Comment by badgerbadgers** [comment]
Styling can be worked on later for sure but the footer component could have be `fixed` or `sticky` footer and will should fix this current footer taking up most of the screen when there is no other content on the page.

Screenshot 2023-05-22 at 10 42 07 PM

**Comment by arigakova** [comment]
> Styling can be worked on later for sure but the footer component could have be `fixed` or `sticky` footer and will should fix this current footer taking up most of the screen when there is no other content on the page.
> 
> Screenshot 2023-05-22 at 10 42 07 PM

I converted footer to sticky footer as you recommended

**Comment by arigakova** [comment]
I converted footer to sticky footer as you recommended

**Comment by badgerbadgers** [comment]
> I converted footer to sticky footer as you recommended

I think it would look better as a fixed footer to maintain consistency with other component design decision like the navbar.

**Review by OxiBo — CHANGES_REQUESTED** [review]
It looks good though I suggest the footer is not fixed positioned. You can put it on the bottom of the screen using classic flex box approach. I advice you create a separate branch for this and create a PR to include it to the main code. Also, please check out the footer elements behavior when reducing the screen size. It might have to be repositioned but it can be taken care later when working with responsive styles.

**Review by OxiBo — APPROVED** [review]

**Pull request merged**